website/content/docs/contribution.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
We can also include a point, in the end, saying if you want to contribute by discussing a possible feature, please consider opening a topic at our mailing list and then provide the mailing list link.